scranShipper/src/App.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+import './App.css'
+import DisplayHomepage from './DisplayHomepage/index.jsx'
+import Footer from "./Footer/Index.jsx";
+import RestaurantDisplay from "./Display/RestaurantDisplay.jsx";
+
+const App = () => {
+
+    const [restaurants, assignRestaurants] = useState([]);
+    useEffect(() => {
+        fetchRestaurants()
+    }, []);
+
+    const fetchRestaurants = () => {
+        fetch('https://food-delivery-api.dev.io-academy.uk/restaurants').then(response => {
+            return response.json();
+        }).then((data) => {
+            assignRestaurants(data);
+        })
+    }
```

Comment:
Why not use async / await syntax instead? It's much nicer than the `.then` version